frontend/src/scenes/settings/environment/SessionRecordingSettings.tsx

```diff
@@ -330,6 +338,121 @@ function LinkedFlagSelector(): JSX.Element | null {
     )
 }
 
+function UrlTriggerForm(): JSX.Element {
+    const { cancelProposingUrlTrigger } = useActions(sessionReplayIngestionControlLogic)
+    const { isProposedUrlTriggerSubmitting } = useValues(sessionReplayIngestionControlLogic)
+
+    return (
+        <Form
+            logic={sessionReplayIngestionControlLogic}
+            formKey="proposedUrlTrigger"
+            enableFormOnSubmit
+            className="w-full flex flex-col border rounded items-center p-2 pl-4 bg-bg-light gap-2"
+        >
+            <div className="flex flex-row gap-2 w-full">
+                <LemonField name="matching">
+                    <LemonSelect options={[{ label: 'Regex', value: 'regex' }]} />
+                </LemonField>
+                <LemonField name="url" className="flex-1">
+                    <LemonInput autoFocus placeholder="Enter URL" data-attr="url-input" />
```

Comment:
we allow in similar components to use `*` as a wildcard. 

do we support that here?

my feeling is we should but happy for that to be a follow-up

Reply:
Regex was the easiest to implement, so I went with that first, but I'll follow up with this